documentation/api/admin/v1/archive.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "PuppetDB 3.0 » Admin API » v1 » Importing and Exporting PuppetDB Archives"
+layout: default
+canonical: "/puppetdb/latest/api/admin/v1/archive.html"
+---
+
+[curl]: ../curl.html#using-curl-from-localhost-non-sslhttp
+
+The `/archive` endpoint can be used for importing and exporting PuppetDB archives.
+
+## `POST /pdb/admin/v1/archive`
+
+This endpoint can be used for streaming a PuppetDB archive into PuppetDB.
+
+### Request Format
+
+The request should be a multipart POST and have `Content-Type: multipart/mixed`.
+
+### URL Parameters
+
+* `archive`: Required. The archive file to import to the PuppetDB.
+* `command_versions`: Required. A JSON object mapping PuppetDB command-names to their version.
```

Comment:
Alright, but it can't really be copy and paste anymore than I have it already, as I gave them the most recent command version, and I'd have to have something a little more complicated in the example to make it version independent
```
tar -xfvz example.tgz
curl -X POST http://localhost:8080/pdb/admin/v1/upload \
       -F "command_versions='$(cat puppetdb-bak/export-metadata.json)'"\
       -F "archive=@example_backup_archive.tgz"
```